### PR TITLE
Add `image2_altRequired` option to `image2` config

### PR DESCRIPTION
**Pull request by andreyfedoseev**
Optionally make the "Alternative Text" field in `image2` plugin required.

### CODE REVIEW

File: plugins/image2/plugin.js [hunk 1]
```diff
@@ -1697,3 +1697,13 @@ CKEDITOR.config.image2_captionedClass = 'image';
  * @cfg {String[]} [image2_alignClasses=null]
  * @member CKEDITOR.config
  */
+
+/**
+ * Determines whether alternative text is required.
+ *
+ *		config.image2_altRequired = false;
+ *
+ * @since 4.5
```
Comment (mlewand) [inline plugins/image2/plugin.js]: Please target it for 4.6.0 instead (it's important to use 3 digs, as we're using semver).

File: plugins/image2/plugin.js [hunk 2]
```diff
@@ -1697,3 +1697,13 @@ CKEDITOR.config.image2_captionedClass = 'image';
...
+ * @cfg {Boolean} [image2_prefillDimensions=false]
```
Comment (mlewand) [inline plugins/image2/plugin.js]: Copy'n'pasta left-over here :) instead `image2_prefillDimensions` we want to use `image2_altRequired`.

Comment (andreyfedoseev) [inline plugins/image2/plugin.js]: @mlewand Thank you for the feedback, I'll update the PR shortly.

**Comment by andreyfedoseev** [comment]
@mlewand I updated the pull requests with tests. I also had to fix a couple of unrelated jshint warnings. Please review.

**Comment by mlewand** [comment]
Good job, I've created a ticket [14889](http://dev.ckeditor.com/ticket/14889).

You made a good job with adjusting code style / creating unit tests, so thanks for quality pull request!

Still there were couple of things to be done, like adding manual test, lang entry meta information for our localization contribution or adding lang entries for other langs - but I did it on our own here.

I've also rebased your branch on top of `major` (CKEditor 4.6.0), since we're done with `4.5.x` line.

To sum it up, your contribution will appear in **4.6.0** version, thank you very much!
